Test-№1.1-from-19.10.2022/task2.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void swap(int *firstElement, int *secondElement)
+{
+    int temp = *secondElement;
+    *secondElement = *firstElement;
+    *firstElement = temp;
+}
+
+void selectionSort(int *array, int lenOfArray)
+{
+    int minim = 0, pointer = 0;

```

Comment:
```suggestion
    int minimum = 0, pointer = 0;

```
Зачем такое странное сокращение? Либо полное слово, либо стандартное сокращение `min`.